addon/lib/field.js

```diff
@@ -330,6 +331,37 @@ export default Base.extend({
     return this.question.isMultipleChoice ? selected : selected[0];
   }),
 
+  /**
+   * The field's JEXL context.
+   *
+   * Properties:
+   * - `form`: Legacy property pointing to the root form.
+   * - `info.form`: The form this question is attached to.
+   * - `info.parent.form`: The parent form if applicable.
+   * - `info.root.form`: The new property for the root form.
+   */
+  jexlContext: computed(
+    "document.jexlContext.{form,info.root.form}",
+    "fieldset.{form.slug,field.fieldset.form.slug}",
+    function() {
+      const context = copy(this.document.jexlContext);
+
+      // This condition is needed to allow for partial states during testing.
+      if (this.get("fieldset")) {
+        context.info.form = this.get("fieldset.form.slug");
+      }
+
+      // Only attach the parent context if in a subform.
+      if (this.get("fieldset.form.slug")) {
```

Comment:
This doesn't really make sense.. The field will always have a fieldset which also will always have a form. This condition will never be false.

I think you want to check whether the fieldset has a `field` property that would mean we are in a subform (the fieldset field is the form question)

Reply:
Good catch! Yeah. Somewhere along the line I must have lost the `fieldset.field` in the first getter.

Reply:
Should be better now. Testing both times for `fieldset.field.fieldset.form.slug`.